assets/style.css

```diff
@@ -0,0 +1,613 @@
+
+.social-share {
+  flex: 1;
+}
+
+.comment-btn {
+  flex: 1.2;
+}
+
+.comment-btn button {
+  width: 100%;
+  height: 40px;
+  font-size: 13px;
+  line-height: 24px;
+  font-weight: 600;
+  padding-top: 6px;
+}
+
+.social-share ul {
+  justify-content: flex-end;
+}
+
+.fixed-footer i {
+  display: inline-block;
+  flex: 1;
+}
+
+.social-share i:hover {
+  background-color: #fff;
+  cursor: pointer;
+}
+
+/* More in Space & Astronomy section */
+
+.extra {
+  width: 85%;
+  margin: auto;
+  grid-template-columns: 3fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  column-gap: 15px;
+  row-gap: 20px;
+  height: 1000px;
+}
+
+.space-astronomy {
+  border-top: 1px solid #000;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: max-content;
+  padding-top: 15px;
+  row-gap: 20px;
+  column-gap: -60px;
+}
+
+.editor-pick {
+  border-top: 1px solid #000;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: max-content 1fr;
+  height: 350px;
+  padding-top: 10px;
+}
+
+.editor-pick h4 {
+  padding: 0;
+  margin: 5px 0;
+  grid-column: 1/-1;
+}
+
+.most-popular h4 {
+  margin-top: 12px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #333;
+  font-weight: 700;
+}
+
+.editor-pick h4:hover {
+  text-decoration: underline;
+}
+
+.most-popular {
+  border-top: 1px solid #000;
+  grid-column: 2/3;
+  grid-row: 1/3;
+  height: 1050px;
+}
+
+/* Footer Section */
+
+.footer {
+  width: 100%;
+}
+
+.movable-footer {
+  width: 80%;
+  margin: 0 auto;
+  border-top: 2px double rgba(180, 178, 178, 0.575);
+  border-top-width: 5px;
+}
+
+.footer-menu ul {
+  padding-inline-start: 0;
+}
+
+.menu {
+  grid-template-columns: repeat(6, 1fr);
+  column-gap: 30px;
+}
+
+.menu li:first-child {
+  font-weight: 700;
+  font-size: 17px;
+  color: #333;
+  margin-top: -10px;
+}
+
+.footer-bottom {
+  border-top: 1px solid rgba(180, 178, 178, 0.575);
+  margin-bottom: 80px;
+}
+
+/* Fixed Footer */
+
+.fixed-footer {
+  font-size: 15px;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  background-color: #fff;
+  border-top: 1px solid #000;
+}
+
+.editor-pick-aside {
+  position: absolute;
+  padding-left: 600px;
+}
+
+.editor-pick-aside p {
+  margin-left: 50px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgb(177, 173, 173);
+  width: 250px;
+  font-weight: 500;
+}
+
+.fixed-footer p:first-child {
+  flex: 4;
+  margin-left: 500px;
+}
+
+.fixed-footer p:nth-child(2) {
+  font-weight: 600;
+  flex: 0.5;
+  text-align: center;
+}
+
+.more-section {
+  padding-right: 30px;
+  border-right: 1px solid rgba(180, 178, 178, 0.575);
+  height: 95%;
+}
+
+.subscribe-section li:nth-child(6),
+.subscribe-section li:nth-child(7),
+.subscribe-section li:nth-child(8),
+.subscribe-section li:nth-child(9),
+.subscribe-section li:nth-child(10),
+.subscribe-section li:nth-child(11) {
+  font-size: 11px;
+}
+
+.subscribe-section li:nth-child(8) {
+  border-bottom: 1px solid rgba(180, 178, 178, 0.575);
+}
+
+.subscribe-section li:nth-child(9) {
+  margin-top: 5px;
+}
```

Comment:
- [ ][MANDATORY] You are making use of inefficient selectors which is bad CSS practice. Please work on this.

Hint: You can create a `font-11` class with rule `.font-11 { font-size: 11px; }` and apply it in your HTML like this `<li class='font-11'>.....<\li>`